recipes/ezomero/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ezomero" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ezomero-{{ version }}.tar.gz
+  sha256: 729db284c27067b8accb127c6501b2205f63e639045339e6939f7c991f0759c7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - omero-py ==5.13.1
```

Comment:
Is this strict pinning required?

Reply:
The version the recipe was prepared for did require that specific pin.
The latest version of ezomero supports a wider range.

Reply:
Let's update to latest unless this one is required.